.gitignore

```diff
@@ -31,3 +31,7 @@ npm_licenses.tar.bz2
 
 # Ignore parser debug
 y.output
+.idea/modules.xml
+.idea/prometheus.iml
+.idea/vcs.xml
+.idea/workspace.xml
```

Comment:
These should go in your global git ignore file:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/